src/routes/jira/server/jira-app-creation-get.ts

```diff
@@ -0,0 +1,19 @@
+import { NextFunction, Request, Response } from "express";
+
+export const JiraAppCreationGet = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<void> => {
+	try {
+		req.log.info("Received Jira app creation page request");
```

Comment:
This should probably be debug or trace

Reply:
I copied the existing log from jira config get: `req.log.info("Received jira configuration page request");` Happy to update though if you think debug/trace is more suitable.

Reply:
Updated the log here and tidied up in other places in this pr: https://github.com/atlassian/github-for-jira/pull/1325